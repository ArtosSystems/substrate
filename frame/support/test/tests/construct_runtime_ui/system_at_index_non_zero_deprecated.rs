use frame_support::construct_runtime;

construct_runtime! {
	pub enum Runtime where
		Block = Block,
		NodeBlock = Block,
		UncheckedExtrinsic = UncheckedExtrinsic
	{
		Balance: balances::{Config, Call, Origin<T>},
		System: system::{Module},
	}
}

fn main() {}
