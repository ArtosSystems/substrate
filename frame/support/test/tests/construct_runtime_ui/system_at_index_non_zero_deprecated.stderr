error: System module must be defined at index `0`, instead Balance is found. (this check is to avoid confusion for generation of origin_caller, (where system is forced to be at index 0 anyway), to bypass this check use attribute `#[deprecated_system_non_zero]` as in `construct_runtime! { #[deprecated_system_non_zero] ... }`).
 --> $DIR/system_at_index_non_zero_deprecated.rs:9:12
  |
9 |         Balance: balances::{Config, Call, Origin<T>},
  |                  ^^^^^^^^
